addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,11 +10976,11 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot for unwatched items (Hides plot and thumb when disabled)"
```

Comment:
Both hiding thumb and hiding plot applies to movies, too? If only hiding plot applies to movies then I think this setting is really inconsistent.
Since I dont really care about this feature I will let other people decide though.